optaplanner-benchmark/src/main/java/org/optaplanner/benchmark/impl/result/ProblemBenchmarkResult.java

```diff
@@ -220,6 +222,20 @@ public boolean hasProblemStatisticType(ProblemStatisticType problemStatisticType
         return false;
     }
 
+    public boolean hasSubSingleCountBiggerThanOne() {
+        if (hasNonDefaultSubSingleCount != null) {
+            return hasNonDefaultSubSingleCount;
```

Comment:
The name of this getter and the boolean field should be the same (as long as they are the same thing which they currently are)